### ISSUE TITLE
does the snickers bar offer still stand

### ISSUE THREAD

**Issue by devinmcnulty** [EricKarsten/Oeconomica-Health-Cohort#4]
[per lesson 8](https://github.com/ekarsten/Oeconomica-Health-Cohort/wiki/Coding-Challenges)

**Comment by devinmcnulty** [comment]
@ekarsten

**Comment by ekarsten** [comment]
Yes, but the size of the snickers bar will scale according to how much of a given challenge the code addresses.

**Comment by devinmcnulty** [comment]
if you write a script that parses texas oil and gas lease addenda:

![image](https://user-images.githubusercontent.com/17799994/43799205-760b9210-9a52-11e8-9e1d-cc65bf7a62d3.png)
